From 1ed69ee2f7c5257296286f6a4832b879c84ce18e Mon Sep 17 00:00:00 2001
From: rlflsdi <111736191+rlflsdi@users.noreply.github.com>
Date: Tue, 23 Aug 2022 15:19:24 +0800
Subject: [PATCH] Create Frontend

---
 Frontend | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 Frontend

diff --git a/Frontend b/Frontend
new file mode 100644
index 0000000..8b13789
--- /dev/null
+++ b/Frontend
@@ -0,0 +1 @@
+
-- 
2.37.2.windows.2

